### PR TITLE
scheduler: remove deprecated v1beta2 KubeSchedulerConfiguration  component config

### PR DESCRIPTION
**Pull request by SataQiu**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
`v1beta2` has been deprecated since v1.25(#111547), it can be removed in v1.28 as planed.

#### Which issue(s) this PR fixes:

Ref #117556

#### Does this PR introduce a user-facing change?

```release-note
kube-scheduler component config (KubeSchedulerConfiguration) kubescheduler.config.k8s.io/v1beta2 is removed in v1.28. Migrate kube-scheduler configuration files to kubescheduler.config.k8s.io/v1.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
-[KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/785-scheduler-component-config-api
```

### CODE REVIEW

File: pkg/scheduler/apis/config/scheme/scheme_test.go [hunk 1]
```diff
@@ -1472,9 +890,9 @@ profiles:
 							{
 								Name: "VolumeBinding",
 								Args: runtime.RawExtension{
-									Object: &v1beta2.VolumeBindingArgs{
+									Object: &v1beta3.VolumeBindingArgs{
```
Comment (kerthcet) [inline pkg/scheduler/apis/config/scheme/scheme_test.go:893]: It should be a mistake before. 😄

**Review by kerthcet — COMMENTED** [review]
/lgtm
Thanks @SataQiu

**Comment by sanposhiho** [comment]
@SataQiu Could you give the reference to https://github.com/kubernetes/kubernetes/issues/117556 on the PR description?

**Comment by SataQiu** [comment]
> Could you give the reference to #117556 on the PR description?

@sanposhiho  Thanks for reminding me. Updated

**Comment by sanposhiho** [comment]
Please update the release note something like:

```
kube-scheduler component config (KubeSchedulerConfiguration) v1beta2 is removed in v1.28. 
You need to migrate to KubeSchedulerConfiguration v1 if you use it.
```

**Comment by SataQiu** [comment]
/assign @liggitt 
for api approve

**Pull request merged**